packages/dds/tree/src/tree/anchorSet.ts

```diff
@@ -17,7 +17,7 @@ export type Anchor = Brand<number, "rebaser.Anchor">;
 /**
  * A singleton which represents a permanently invalid location (i.e. there is never a node there)
  */
-const NeverAnchor: Anchor = brand(0);
+export const NeverAnchor: Anchor = brand(0);
```

Comment:
Exposing this publicly makes it no longer an implementation detail of this particular anchorSet implementation, and makes details about how its lifetime works (ex: that you don't have to free it) part of the public API.

We would prefer to not do this to retain the ability to change how this works in the future, and keep the APi smaller.

Instead, users which know they don't need an anchor should use some other value (such as undefined) when they don't have an anchor, and if you really need an anchor which will never resolve, you can pass "null" to AnchorSet.track (and pass it to forget when done with it: documentation currents says thats required, and future versions might leak otherwise)